src/test/components/Details.test.js

```diff
@@ -22,14 +23,6 @@ describe('app/DetailsContainer', function() {
     E E
   `);
 
-  const tabSlugs: TabSlug[] = [
-    'stack-chart',
-    'marker-chart',
-    'flame-graph',
-    'marker-table',
-    'calltree', // 'calltree' not first on purpose
```

Comment:
If you wonder about this, that "calltree" will be back first, the comment wasn't uptodate: it isn't really useful now that we generate a new store for each slug (see below).

Same for the next file too.